### DISCUSSION TITLE
Plain text correction with Inception

### DISCUSSION THREAD

**Discussion by albinasarymsakova** [inception-project/inception#4170] [Q&A]
Hey! Thank you for this fantastic tool! 

I have a question. Is it possible to change a plain text (from txt you’ve uploaded previously) using Inception in case you have erroneous tokens or something before starting the annotation? If it is, would you mind explaining how to do that? 

Thank everybody in advance!

**Comment by reckart** [comment]
The text cannot be changed after import. Also, token boundaries can currently not be changed after import.

You would need to fix your text before importing into INCEpTION. 

You can import using a format like `Plain text (space-separated tokens, one sentence per line)` if you need more control over token/sentence boundaries.

**Reply by albinasarymsakova** [reply]
Thank you very much, Richard!
